core/workflow-operator/src/test/scala/edu/uci/ics/amber/operator/stablemergesort/StableMergeSortOpExecSpec.scala

```diff
@@ -0,0 +1,302 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package edu.uci.ics.amber.operator.stablemergesort
+
+import edu.uci.ics.amber.core.tuple.{Attribute, AttributeType, Schema, Tuple}
+import edu.uci.ics.amber.operator.stablemergesort.StableMergeSortOpDesc.StableSortKey
+import edu.uci.ics.amber.util.JSONUtils.objectMapper
+import org.scalatest.flatspec.AnyFlatSpec
+
+import java.sql.Timestamp
+import scala.jdk.CollectionConverters._
+
+class StableMergeSortOpExecSpec extends AnyFlatSpec {
+
+  private def schemaOf(attributes: (String, AttributeType)*): Schema = {
+    attributes.foldLeft(Schema()) { case (acc, (name, tpe)) => acc.add(new Attribute(name, tpe)) }
+  }
```

Comment:
This might be a good method. We can open another PR add it as a Schema object function.